.github/workflows/rust.yml

```diff
@@ -37,7 +37,7 @@ permissions:
 jobs:
   test:
     runs-on: ubuntu-latest-16-cores
-    timeout-minutes: 45
+    timeout-minutes: 90
```

Comment:
That is a bit sad

Reply:
Well typically this time it passed in under 45 😅 I guess it was nearly done last time when it got cancelled.

Reply:
Back to 45 now.  The cache should catch it.